mw-webapp/src/App.tsx

```diff
@@ -1,14 +1,28 @@
+import {useEffect, useState} from "react";
+import {getAuth, User} from "firebase/auth";
 import {Header} from "src/component/header/Header";
 import {Router} from "src/router/Router";
+import {handleUserAuthState} from "src/service/auth/handleUserAuthState";
 
 /**
  * App
  */
 export const App = () => {
+  const [user, setUser] = useState<User | null>(null);
+  const isAuth = getAuth();
+  useEffect(() => {
+    handleUserAuthState(setUser);
+  }, []);
+
+  useEffect(() => {
+    if (isAuth.currentUser) {
+      localStorage.setItem("auth", "true");
+    }
+  }, [isAuth.currentUser]);
 
```

Comment:
Why do you need 2 useEffect in one file?

Reply:
One useEffect work when first render page, second useEffect work when chnage auth object (register user)

Reply:
Because one useEffect work when first render app, second useEffect work when change auth object